airflow/contrib/hooks/salesforce_hook.py

```diff
@@ -18,127 +18,110 @@
 # under the License.
 #
 """
-This module contains a Salesforce Hook
-which allows you to connect to your Salesforce instance,
-retrieve data from it, and write that data to a file
-for other uses.
+This module contains a Salesforce Hook which allows you to connect to your Salesforce instance,
+retrieve data from it, and write that data to a file for other uses.
 
-NOTE:   this hook also relies on the simple_salesforce package:
-        https://github.com/simple-salesforce/simple-salesforce
+.. note:: this hook also relies on the simple_salesforce package:
+      https://github.com/simple-salesforce/simple-salesforce
 """
-from simple_salesforce import Salesforce
-from airflow.hooks.base_hook import BaseHook
-
-import json
+import time
 
 import pandas as pd
-import time
+from simple_salesforce import Salesforce
 
+from airflow.hooks.base_hook import BaseHook
 from airflow.utils.log.logging_mixin import LoggingMixin
 
 
 class SalesforceHook(BaseHook):
-    def __init__(
-            self,
-            conn_id,
-            *args,
-            **kwargs
-    ):
+
+    def __init__(self, conn_id):
         """
-        Create new connection to Salesforce
-        and allows you to pull data out of SFDC and save it to a file.
+        Create new connection to Salesforce and allows you to pull data out of SFDC and save it to a file.
+
+        You can then use that file with other Airflow operators to move the data into another data source.
 
-        You can then use that file with other
-        Airflow operators to move the data into another data source
+        :param conn_id: the name of the connection that has the parameters we need to connect to Salesforce.
+                        The connection should be type `http` and include a user's security token
```

Comment:
This is indented too far for docstring rendering? see https://github.com/apache/airflow/pull/4895#discussion_r268388705

Reply:
Oh really? Did not know that. I have not checked the docs on mobile. Will fix this.